docs/process/roadmap.md

```diff
@@ -1,63 +1,70 @@
 # Roadmap
 
-The following are our marketing releases as planned to date. They contain the broad features we're considering. The further away a release is, the less defined it is. Releases are subject to change at any time.
+The following are our marketing releases, as planned to date (and we plan to revisit 6 months from now). They contain the broad features we're considering. The further away a release is, the less defined it is. Releases are subject to change at any time. This is not written in stone!!!
 
-## 1.1
+## 1.2 - Relationship between branches - lite
 
-- Relationship to _X_
-  - Show how the current branch relates to the target branch
-  - The target is the default branch or the upstream's default branch
-  - _Probably_ let the user switch the target branch
+- [Relationship between branches lite](https://github.com/desktop/desktop/issues/3956)
+  - Compare my work to any branch
+  - Merge in any other branch to my branch
+  
+- [Upgrade nudge from classic to the new app](https://github.com/desktop/desktop/issues/2146)
 
-- Conflict resolution lite
-  - Differentiate conflicted files from other changed files
-  - Let me choose my mergetool
-  - Let me open my mergetool
+## 1.3 - Detecting marge conflicts - 2.0
```

Comment:
> marge conflicts

![image](https://user-images.githubusercontent.com/1174461/36824086-ad40510c-1ca4-11e8-9f0e-e135fec4040c.png)

Reply:
Can we just leave this as-is? :trollface:

Reply:
But more seriously, what's the `2.0` part about this represent? Conflict resolution **lite** is set down for `1.6` - should this be `1.0` instead?